webapp/channels/src/components/drafts/drafts.tsx

```diff
@@ -46,7 +70,22 @@ function Drafts({
         return () => {
             dispatch(unsuppressRHS);
         };
-    }, []);
+    }, [dispatch]);
+
+    const handleSwitchTabs = useCallback((key) => {
+        switch (key) {
+        case 0:
+            if (!isDraftsTab) {
+                history.push(`/${match.params.team}/drafts`);
```

Comment:
To avoid needing the `match.params.team`... would a relative url work here?

Reply:
hey, so I tried doing that but it just doesn't work in react router v5. Skipping this for now.